shell/server/core.js

```diff
@@ -501,16 +518,27 @@ restoreInternal = (originalToken, ownerPattern, requirements, tokenId, saveByCop
       // discard persistentMethods.
```

Comment:
`persistentMethods` is no longer a thing, so this comment doesn't make sense. It should probably refer to `PersistentImpl` instead.